kubernetes/config/common.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+data:
+  common.libsonnet: |
+    {
+      blobstore: {
+        contentAddressableStorage: {
+          sharding: {
+            hashInitialization: 11946695773637837490,
+            shards: [
+              {
+                backend: { grpc: { address: 'storage-0.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+              {
+                backend: { grpc: { address: 'storage-1.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+            ],
+          },
+        },
+        actionCache: {
+          sharding: {
+            hashInitialization: 14897363947481274433,
+            shards: [
+              {
+                backend: { grpc: { address: 'storage-0.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+              {
+                backend: { grpc: { address: 'storage-1.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+            ],
+          },
+        },
+      },
+      browserUrl: 'http://localhost:7980',
+      maximumMessageSizeBytes: 16 * 1024 * 1024,
+      httpListenAddress: ':7980'
```

Comment:
What's the reason we're moving away from `:80` here? For the docker-compose based setup, it makes sense. Eventually we want to expose all of the services on the system itself, so people can debug. For a system like Kubernetes, every pod gets its own IP address. There is therefore no need to distinguish between multiple port numbers, right?